lib/datagrid/drivers/mongoid.rb

```diff
@@ -64,7 +64,7 @@ def normalized_column_type(scope, field)
         return nil unless type
         {
           [BigDecimal , String, Symbol, Range, Array, Hash, ] => :string,
-          [Boolean] => :boolean,
+          [::Mongoid::Boolean] => :boolean,
```

Comment:
Is `Mongoid::Boolean` defined for older version of mongoid? Maybe it is better to check like this:

```suggestion
          [defined?(::Mongoid::Boolean) && ::Mongoid::Boolean, defined?(Boolean) && Boolean] => :boolean,
```

Reply:
`Mongoid::Boolean` exists since 2013, that is also when `Boolean` was removed: https://github.com/mongodb/mongoid/commit/1424f86015a54c700207e875d00a03acfa9370ff

`Boolean` doesn’t work for me:
```
Failures:

  1) Datagrid::Filters::DateFilter with orm mongoid date to timestamp conversion when single date paramter given
     Failure/Error: [Boolean] => :boolean,

     NameError:
       uninitialized constant Datagrid::Drivers::Mongoid::Boolean

                 [Boolean] => :boolean,
                  ^^^^^^^
     # ./lib/datagrid/drivers/mongoid.rb:67:in `normalized_column_type'
```